docs/topics/install-eap-plugin.md

```diff
@@ -24,3 +24,10 @@ You can follow these instructions to install [the preview version of the Kotlin
 4. Click **Install**. 
 
 If you want to work on existing projects that were created before installing the EAP version, you need to [configure your build for EAP](configure-build-for-eap.md). 
+
+## If you run into any problems
+
+* Report issues to [our issue tracker, YouTrack](https://kotl.in/issue).
+* Find help in the _#eap_ channel on [Slack](https://kotlinlang.slack.com/) ([get an invite](https://surveys.jetbrains.com/s3/kotlin-slack-sign-up)).
+* Roll back to the latest stable version by selecting the **Stable** update channel in 
```

Comment:
This is a good one, but we also can get rid of:
 * **Gerund**: Roll back to the latest stable version – In **Tools | Kotlin | Configure Kotlin Plugin Updates** select the **Stable** update channel and click **Install**.
* **Repetition**: **Install** to install, roll back to the lastest stable ... to install the latest version.

Reply:
Thanks! Updated. I'll proofread the new sentence and then merge.